_includes/meeting.html

```diff
@@ -1,4 +1,4 @@
-{% assign meeting_date = meeting.date | date: "%A %-dth %B %Y" | replace: "1th", "1st" | replace: "2th", "2nd" | replace: "3th", "3rd" | replace: "11st", "11th" | replace: "12nd", "12th" | replace: "13rd", "13th" %}
+{% assign meeting_date = meeting.date | date: "%A %-dth %B %Y 18:30 PM" | replace: "1th", "1st" | replace: "2th", "2nd" | replace: "3th", "3rd" | replace: "11st", "11th" | replace: "12nd", "12th" | replace: "13rd", "13th" %}
```

Comment:
This looks good!

I wonder how we should format the full date and time string though: should the time go _before_ the date? Should it be separated by a comma? Should it use a 24 hour or 12 hour clock?

Perhaps we could copy the format used on the [Attending.io pages](https://attending.io/events/london-computation-club-ch-16-metatheory-of-subtyping)?

> 6:30 PM-8:30 PM Tuesday 19 September 2017

Reply:
I've added the start and end time at the beginning so the next one will display as: 

> 6:30 PM-8:30 PM Tuesday 21st November 2017